build-system/tasks/csvify-size/index.js

```diff
@@ -174,6 +176,22 @@ function mergeTables(dateTimes, tables) {
 }
 
 /**
+ * @param {string} prettyBytes
+ * @return {number}
+ */
+function reversePrettyBytes(prettyBytes) {
+  var triple = prettyBytes.match(/(\d+(?:\.\d+)?)\ (B|kB|MB|GB|TB|PB|EB|ZB|YB)/);
```

Comment:
Why is `kB` lowercase but everything else is uppercase?

LOL, are we gonna push TB files?!